phpmyadmin/DOCS.md

```diff
@@ -30,6 +30,11 @@ log_level: info
 
 **Note**: _This is just an example, don't copy and paste it! Create your own!_
 
+### Option: `upload_limit`
+
+By default the size limit for uploads (for operations such as imports) is set to
+64MB.  This can be increased with this option, for example `100` would be 100MB.
```

Comment:
```suggestion
By default, the size limit for uploads (for operations such as imports) is set to
64MB. This can be increased with this option, for example, `100` would be 100MB.
```